weaver/main.go

```diff
@@ -87,6 +89,18 @@ func main() {
 		routerName = iface.HardwareAddr.String()
 	}
 
+	if hostName == "" {
+		osHostName, err := os.Hostname()
+		if err != nil {
+			log.Fatal(err)
+		}
+		if osHostName == "" {
+			hostName = "<unknown>"
```

Comment:
I reckon that will look rather odd in the various places where that gets displayed. Set it to "", perhaps?